<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../rsc.qrc">
    <normaloff>:/icones/Images/Icones/iconeSortingAlg.png</normaloff>:/icones/Images/Icones/iconeSortingAlg.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>50</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">qproperty-alignment: AlignCenter;</string>
      </property>
      <property name="text">
       <string>sorting algorithms</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widgetGifs" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="labelInsertion">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelSelection">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelBubble">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelMerge">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelQuick">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelHeap">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widgetText" native="true">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_textInsertion">
         <property name="text">
          <string>insertion</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_textSelection">
         <property name="text">
          <string>selection</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_textBubble">
         <property name="text">
          <string>bubble</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_textMerge">
         <property name="text">
          <string>merge</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_textQuick">
         <property name="text">
          <string>quick</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_textHeap">
         <property name="text">
          <string>heap</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>818</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New_odenation"/>
   </widget>
   <widget class="QMenu" name="menuStatistics">
    <property name="title">
     <string>&amp;Statistics</string>
    </property>
    <addaction name="actionGraph"/>
   </widget>
   <widget class="QMenu" name="menu_Maintenance">
    <property name="title">
     <string>&amp;Maintenance</string>
    </property>
    <addaction name="action_Data_base"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuStatistics"/>
   <addaction name="menu_Maintenance"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New_odenation"/>
   <addaction name="separator"/>
   <addaction name="actionGraph"/>
   <addaction name="separator"/>
   <addaction name="action_Data_base"/>
  </widget>
  <action name="action_New_odenation">
   <property name="icon">
    <iconset resource="../rsc.qrc">
     <normaloff>:/icones/Images/Icones/iconeSort.png</normaloff>:/icones/Images/Icones/iconeSort.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New odenation</string>
   </property>
  </action>
  <action name="actionGraph">
   <property name="icon">
    <iconset resource="../rsc.qrc">
     <normaloff>:/icones/Images/Icones/iconeGraph.png</normaloff>:/icones/Images/Icones/iconeGraph.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Graph</string>
   </property>
  </action>
  <action name="action_Data_base">
   <property name="icon">
    <iconset resource="../rsc.qrc">
     <normaloff>:/icones/Images/Icones/iconeDataBase.png</normaloff>:/icones/Images/Icones/iconeDataBase.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Data base</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../rsc.qrc"/>
 </resources>
 <connections/>
</ui>
